agent/trufflehog_agent.py

```diff
@@ -105,6 +106,10 @@ def _report_vulnz(self, vulnz: list[dict[str, Any]], message: m.Message) -> None
                     risk_rating=vuln_mixin.RiskRating.HIGH,
                     technical_detail=technical_detail,
                     vulnerability_location=vulnerability_location,
+                    dna=_compute_dna(
+                        vuln_title=kb.KB.SECRETS_REVIEW.title,
```

Comment:
You need to add the secret_token in the DNA computation otherwise two secrets in the same file will have the same DNA!

Reply:
Fixed